keras/applications/convnext.py

```diff
@@ -217,8 +218,11 @@ def __init__(self, init_values, projection_dim, **kwargs):
         self.projection_dim = projection_dim
 
     def build(self, input_shape):
-        self.gamma = tf.Variable(
-            self.init_values * tf.ones((self.projection_dim,))
+        self.gamma = self.add_weight(
+            shape=(self.projection_dim,),
+            dtype=self._compute_dtype_object,
```

Comment:
The dtype is not needed since it will be inferred from the initializer.

Reply:
Ok, no worries. Made a new commit removing the dtype.